eng/WpfArcadeSdk/tools/Pbt.props

```diff
@@ -46,18 +43,4 @@
         as a convenient 'transport package').
   -->
 
-  <!--
-    If local PresentationBuildTasks project is present, then import WinFX.props from local sources;
-    otherwise import Microsoft.NET.Sdk.WindowsDesktop.props from Microsoft.NET.Sdk.WindowsDesktop
-
-    We can not really test for $(InternalMarkupCompilation)==true here. It is usually defined
-    in the csproj - which is included after props (but before targets).
-  -->
-  <Import Project="$(WpfSourceDir)PresentationBuildTasks\Microsoft.WinFX.props"
-          Condition="Exists('$(LocalMicrosoftWinFXProps)') "/>
-
-  <Import Sdk="Microsoft.NET.Sdk.WindowsDesktop"
```

Comment:
I am not very sure deleting that is safe. However, after the last change, Microsoft.WinFX.props files is empty

Reply:
Did you confirm that this is still picking up the locally built version of PresentationBuildTasks?

Reply:
This can only be tested by flowing (simulating the flow) the wpf.arcade package to the wpf-int repo, and building a project there that relies on InternalMarkupCompilation. One may not exist there today, but creating a test app that does should be quite easy.

Reply:
I now see that InternalMarkupCompilation isn't set in the WPF repo.

Reply:
Sorry, I have no idea what you are talking about. What is the action item here?  Can I remove this or not? I could left the empty WinFXProps file if we need it

Reply:
@wli3:  Can you add the replacement of pbt.props/targets in to your refactoring work issue?  Thanks.

Reply:
Do you have doc for what pbt.props/targets is for? I did read the binlog of build of wpf repo. But I still don't have a clue

Reply:
There is some description in the markdown file.  Sorry, I don't believe there is a separate doc with a more detailed description.  We can discuss it during the meeting later today.  Thanks.

Reply:
Do you have any on boarding document that I can read in the meanwhile?